Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -248,6 +286,14 @@ export default class MedicationTimelineComponent extends Vue {
     return this.medicationLoaded && this.entry?.pharmacy?.isLoaded;
   }
 
+  private get commentsVisible(): boolean {
```

Comment:
this is redundant since is just returning a simple boolean. Lets remove this get methods and use the variable in the html